token/core/fabtoken/v1/driver/driver.go

```diff
@@ -100,6 +100,15 @@ func (d *Driver) NewTokenService(tmsID driver.TMSID, publicParams []byte) (drive
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to initiliaze public params manager")
 	}
+	// log information about the public params
+	pp := publicParamsManager.PublicParams()
+	logger.Infof("new token driver for tms id [%s] with label and version [%s:%s]", tmsID, pp.Identifier(), pp.Version())
+	for _, issuer := range pp.Issuers() {
```

Comment:
You can do
```
logger.Debug("issuers are [%s]", pp.Issuers())
```

Reply:
I wanted to have them on separated lines, for clarify.

Reply:
removed, I'll just print the public params.